assets/src/edit-story/components/library/panes/text/textTab.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import useLibrary from '../../useLibrary';
+import { Tab } from '../shared';
+import paneId from './paneId';
+import { ReactComponent as TextIcon } from './text.svg';
+import { ReactComponent as TextAddIcon } from './text_add.svg';
+
+const AnimatedTextIcon = styled(TextIcon).attrs(({ isSecondary }) => ({
+  style: {
+    // scales 28px to 20px
+    transform: isSecondary ? 'scale(0.71)' : 'none',
+  },
+}))``;
+
+const AnimatedTextAddIcon = styled(TextAddIcon).attrs(({ isPrimary }) => ({
+  style: {
+    // scales 28px to 48px
+    transform: isPrimary ? 'scale(1.71)' : 'none',
+  },
+}))``;
+
+const QuickAction = styled.button`
+  margin: 0 0 0 -10px;
+  background: transparent;
+  border: 0;
+  padding: 0;
+  line-height: 1;
+  overflow: hidden;
+  cursor: pointer;
+  color: ${({ theme }) => theme.colors.fg.v1};
+`;
+
+function TextTab(props) {
+  const {
+    actions: { insertElement },
+  } = useLibrary();
+
+  const [isHoveringQuick, setIsHoveringQuick] = useState(false);
+  const [isFocusingQuick, setIsFocusingQuick] = useState(false);
+
+  const handleAddText = (evt) => {
+    evt.stopPropagation();
+    evt.preventDefault();
```

Comment:
Are both `stopPropagation ` and `preventDefault` needed here?

Reply:
Good point. `preventDefault` isn't strictly necessary here, removed in a24838a0 along with measures to remove some console errors arising from unwanted props leaking through to components. Thanks for flagging!